content/22.vaccines.md

```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
 
 ### Live-Attenuated Viruses
 
-<!--To Do: Not used this time I think?-->
-Although this is the vaccine development strategy with the longest history... (short paragraph explaining not really in play here)
+Live-attenuated vaccines (LAV) or replication-competent vaccines use a weakened living version of a disease-causing virus or a version of a virus that is similar to induce an immune response [@doi:10.1038/s41586-020-2798-3]. 
+The virus can be attenuated by passinging it in a foreign host, until as a consequence of selection pressure the virus loses its efficacy in the original host. 
+Alternatively, selective gene deletion or codon de-optimization can be utilized to attenuate the virus [@doi:10.1038/s41586-020-2798-3]. 
+LAVs are used globally to prevent diseases caused by viruses such as measles, rubella, polio, influenza, varicella zoster, and the yellow fever virus [@doi:10.1016/B978-0-323-35761-6.00002-X].
+It is generally recognized that LAVs induce an immune response that is similar to natural infection and are favoured because they induce long-lasting and robust immunity that can protect from disease. 
+This is in part as a result of the immune response to the range of viral antigens available that tend to be more immunogenic than those from non-replicating vaccines [@url:https://www.hhs.gov/immunization/basics/types/index.html; @doi:10.1016/j.virol.2015.03.032; @doi:10.3389/fimmu.2020.602256]. 
+LAVs are also favoured because they tend to be restricted to viral replication in the tissues around the location of inoculation [@doi:10.1128/9781555818951] and some can be administered intranasally [@doi:10.1038/s41586-020-2798-3]. 
+
+The first deliberate attempt to utilize an attenuated viral vaccine dates back to Louis Pasteur in 1885, despite not knowing that the disease-causing agent he was experimenting with was a virus. 
+Indeed, the next intentional LAVs developed were intended to prevent illness caused by the yellow fever virus in 1935, followed by the first influenza vaccine in 1936 [@doi:10.1128/9781555818951]. 
+Although LAV development strategies have the longest history, this strategy has not been widely utilized against SARS-CoV-2 and COVID-19. 
+There is general concern that LAV strategies may risk causing disease in individuals who are immunocompromised [@doi:10.1016/j.copbio.2007.10.010], which is an even greater concern when dealing with a novel virus and disease. 
+Previously, there have been numerous attempts to develop both SARS-CoV and MERS-CoV LAVs [@doi:10.3389/fimmu.2020.602256], but no vaccine has been approved for use in humans.
+Nowadays, the manufacture of LAVs use safe and reliable methods to produce large quantities of vaccines once they have undergone rigorous pre-clinical studies and clinical trials to evaluate their safety and efficacy.
```

Comment:
There is a mention of the polio outbreak in the intro, so I think maybe add a clause to hearken back to that?
```suggestion
While safety in production was a major concern in the past, nowadays manufacturers of LAVs use safe and reliable methods to produce large quantities of vaccines once they have undergone rigorous preclinical studies and clinical trials to evaluate their safety and efficacy.
```